lib/core/src/persist/mod.rs

```diff
@@ -312,11 +312,14 @@ impl Persister {
                         preimage: maybe_send_swap_preimage,
                         bolt11: maybe_send_swap_invoice.clone(),
                         payment_hash: maybe_send_swap_payment_hash,
-                        description: maybe_send_swap_description.unwrap_or_else(|| {
-                            maybe_send_swap_invoice
-                                .and_then(|bolt11| get_invoice_description!(bolt11))
-                                .unwrap_or("Lightning payment".to_string())
-                        }),
+
+                        description: "Lightning payment".to_string(),
+                        // TODO Should not assume this is bolt11
+                        // description: maybe_send_swap_description.unwrap_or_else(|| {
+                        //     maybe_send_swap_invoice
+                        //         .and_then(|bolt11| get_invoice_description!(bolt11))
```

Comment:
Maybe it is safe to remove this now, since 4 months the invoice description (if available) is being added to the persisted swap

Reply:
Good point, done: 790d8f205658468be994a57c67220f75ab0fbb49